src/trace_context.js

```diff
@@ -68,11 +68,25 @@ function RunInContext(createTracer, handle) {
 
 
 /**
- * Returns the active trace
- * @return {Object} tracer object
+ * Gets trace reference from executionAsyncId or from epsagonIdentifier.
+ * @param {*} epsagonIdentifier gets the reference from epsagonIdentifier
+ * @returns {*} trace reference
  */
-function get() {
-    return tracers[asyncHooks.executionAsyncId()] || null;
+function get(epsagonIdentifier) {
+    if (tracers[asyncHooks.executionAsyncId()]) {
+        return tracers[asyncHooks.executionAsyncId()];
+    } if (epsagonIdentifier) {
+        return tracers[epsagonIdentifier];
+    }
+    return null;
+}
+
+/**
+ * Creates a reference to epsagonIdentifier
+ * @param {Number} epsagonIdentifier sets the reference to this epsagonIdentifier
+ */
+function setTraceToEpsagonId(epsagonIdentifier) {
+    tracers[epsagonIdentifier] = get(epsagonIdentifier);
```

Comment:
It seems like this should never work because when you set it, `get(epsagonIdentifier)` should be null